docs/proxy/datacenter_proxy/examples.md

```diff
@@ -120,7 +118,10 @@ const Apify = require("apify");
 
 Apify.main(async () => {
     const requestList = await Apify.openRequestList(
-        "my-list", ["http://www.example.com"]
+        "my-list", [
+            "http://proxy.apify.com/?format=json",
+            "http://proxy.apify.com/?format=json",
```

Comment:
This won't work because the URLs will be deduplicated so only one page will be opened. You can use this: https://sdk.apify.com/docs/typedefs/request-list-options#keepduplicateurls but I think it would just complicate the example. You can use it once with and once without the `format=json`. Those would not be deduplicated.